src/components/Limit.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<div class="querybuilder__limit">
+		<input
+			class="querybuilder__limit-checkbox"
+			type="checkbox"
+			name="limit"
+			v-model="checked"
+		/>
+		<label for="limit">
```

Comment:
the `for` attribute refers to an id, not a name, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for

Reply:
The other PR will be eventually merged here.

Reply:
This will be replaced as the label, and even the component are going to be replaced. Will update it but I dont think it's worth spending much time on this part, tomorrow it will be probably decided how to handle this.

Reply:
If the idea is to merge #120 into this PR first, then I'm ok with it :)

Reply:
Oh, no. The idea was to merge this one into [PR 118](https://github.com/wmde/query-builder/pull/118). I've updated this PR